File: Cpp_solving/exercise_08.cpp
/*
    Given a triangle represented by its sides L1, L2, L3, determine and print a legend as appropriate:
    equilateral, isosceles or scalenes.
*/

#include <iostream>
using namespace std;

void wich_triangle (int side_1, int side_2, int side_3){
    if ((side_1 == side_2 && side_1 != side_3) || 
        (side_2 == side_3 && side_2 != side_1) || 
        (side_3 == side_1 && side_3 != side_2)){
        cout << "Its an isosceles triangle!";
    }
    else if (side_1 != side_2 && side_2 != side_3 && side_3 != side_1){
        cout << "Its a scalenes triangle!";
    }
    else cout << "Its an equilateral triangle!";
}

int main () {
    wich_triangle(1,1,2);
 return 0;
}

File: Python_solving/exercise_08.py
# Given a triangle represented by its sides L1, L2, L3, determine and print a legend as appropriate:
#  equilateral, isosceles or scalenes.
def wich_triangle(side1,side2,side3):
    if ((side1 == side2 != side3) or 
        (side2 == side3 != side1) or 
        (side3 == side1 != side2)):
        print("They form a isosceles!")
    elif (side1 == side2 == side3):print("They form a equilateral")
    else: print("They form a scalenes")
    
wich_triangle(1,3,2)